### PR TITLE
Remove `is` prefix in `@yamada-ui/autocomplete`

### LINKED ISSUE #3438: Rename props: `is*` eliminates all is prefix in `@yamada-ui/autocomplete`
### Which package has an issue?

@yamada-ui/react

### Package Version

2.0

### Description

Change and unify the names of props.
Removing all is prefix from `is*`

### Link to Reproduction

_No response_

### Steps to Reproduce

_No response_

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

Yes

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by cidkumagai**
Related #3438

### CODE REVIEW

**Review by 108yen — APPROVED** [review]

**Comment by kaehehehe** [comment]
Hi @cidkumagai. I have a question. Do we need a changeset file? I got a comment to add one to [my PR](https://github.com/yamada-ui/yamada-ui/pull/3442), and I'm a bit confused because other related PRs don't have a changeset file.

File: packages/components/autocomplete/src/multi-autocomplete.tsx [hunk 1]
```diff
@@ -150,6 +158,7 @@ export const MultiAutocomplete = forwardRef<MultiAutocompleteProps, "input">(
     )
     let {
       className,
+      clearable = true,
```
Comment (hirotomoyamada) [inline packages/components/autocomplete/src/multi-autocomplete.tsx:161]: Change `clearable = true` to `clearable`.

File: packages/components/autocomplete/src/multi-autocomplete.tsx [hunk 2]
```diff
@@ -175,7 +184,7 @@ export const MultiAutocomplete = forwardRef<MultiAutocompleteProps, "input">(
       portalProps = { isDisabled: true },
       ...computedProps
     } = omitThemeProps(mergedProps)
-
+    clearable = clearable && isClearable
```
Comment (hirotomoyamada) [inline packages/components/autocomplete/src/multi-autocomplete.tsx:187]: Remove this line.

File: packages/components/autocomplete/src/multi-autocomplete.tsx
Comment (hirotomoyamada) [inline packages/components/autocomplete/src/multi-autocomplete.tsx:1]: Insert `clearable ??= isClearable` after the line `minH ??= minHeight`.

File: packages/components/autocomplete/src/multi-autocomplete.tsx (hunk 1 above)
Comment (cidkumagai) [reply packages/components/autocomplete/src/multi-autocomplete.tsx]: fixed.

File: packages/components/autocomplete/src/multi-autocomplete.tsx (hunk 2 above)
Comment (cidkumagai) [reply packages/components/autocomplete/src/multi-autocomplete.tsx]: fixed.

File: packages/components/autocomplete/src/multi-autocomplete.tsx
Comment (cidkumagai) [reply packages/components/autocomplete/src/multi-autocomplete.tsx:1]: fixed.

**Comment by cidkumagai** [comment]
@hirotomoyamada 
Please re-review!

**Review by hirotomoyamada — DISMISSED** [review]

**Comment by Shunii85** [comment]
@cidkumagai @hirotomoyamada 
Don't you need a changeset file?

**Comment by hirotomoyamada** [comment]
@Shunii85 @cidkumagai 

That's right. Please add.

**Comment by cidkumagai** [comment]
@hirotomoyamada  @Shunii85 
Added changeset.
please re-review!

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**